origin.spec

```diff
@@ -82,11 +82,14 @@ install -m 0644 -t %{buildroot}%{_sysconfdir}/sysconfig %{SOURCE10}
 
 mkdir -p %{buildroot}/var/log/%{name}
 
+ln -s %{_bindir}/openshift %{_bindir}/osc
```

Comment:
This needs to be, otherwise it tries to write the symlink to the root of the host building the rpm instead of the relative root of the buildroot:
```
ln -s %{_bindir}/openshift %{buildroot}%{_bindir}/osc 
```